pkg/gen/python-templates/helm/v2/helm.py

```diff
@@ -323,7 +323,8 @@ def _parse_chart(all_config: Tuple[str, Union[ChartOpts, LocalChartOpts], pulumi
 
         # Fetch the chart.
         _fetch(chart_to_fetch, fetch_opts)
-        fetched_chart_name = os.listdir(chart_dir)[0]
+        # Sort the directories into alphabetical order, and choose the first
+        fetched_chart_name = sorted([f for f in os.listdir(chart_dir)], key=str.lower)[0]
```

Comment:
`os.listdir` already returns an array.

```suggestion
        fetched_chart_name = sorted(os.listdir(chart_dir), key=str.lower)[0]
```

Reply:
Good catch, resolved